hr_attendance_report_theoretical_time/tests/test_hr_attendance_report_theoretical_time.py

```diff
@@ -0,0 +1,444 @@
+# Copyright 2017-2019 Tecnativa - Pedro M. Baeza
+# Copyright 2021 Landoo Sistemas de Informacion SL
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+import datetime
+
+from odoo.addons.base.tests.common import BaseCommon
+
+
+class TestHrAttendanceReportTheoreticalTimeBase(BaseCommon):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.env(
+            context=dict(
+                cls.env.context,
+                mail_create_nolog=True,
+                mail_create_nosubscribe=True,
+                mail_notrack=True,
+                no_reset_password=True,
+                tracking_disable=True,
+            )
```

Comment:
This should be removed when using BaseCommon

Reply:
All changes have been made and the administrative commits are squashed